doc/_static/js/custom.js

```diff
@@ -26,17 +26,30 @@ $(document).ready(function () {
     // TODO: this should listen for click events in the whole terminal
     // window, not only in the content sub-window
     // Clicking on the terminnals copies the content to the clipboard
-    $("div.highlight").click(function () {
-        if ("jquery: ", $(this).parent().attr('class') == "doctest highlight-default notranslate") {
-            let text = $(this).text();
+    $("[class*='highlight-']") .click(function () {
+        const formatCopiedText = (text) => {
             textArr = text.split("\n");
             textArr = textArr.filter(line => (line.startsWith("... ") || line.startsWith(">>> ")));
             text = textArr.join("\n");
             text = text.replaceAll("... ", "").replaceAll(">>> ", "");
-            navigator.clipboard.writeText(text);
-        } else {
-            navigator.clipboard.writeText($(this).text());
+            return text;
         }
+
+        // https://stackoverflow.com/questions/31128855/comparing-ecma6-sets-for-equality
+        const eqSet = (xs, ys) => 
+            xs.size === ys.size &&
+            [...xs].every((x) => ys.has(x));
+
+        const terminalText = $(this).find('div.highlight').text();
+        const isPythonConsole = eqSet(new Set($(this).attr('class').split(' ')),
```

Comment:
isPythonConsole - Are you sure this comparison is ever true? The term 'doctest' didn't turn up anything on my search.

Reply:
Yeah, like for this page, the terminals have the classes: https://docs.ploomber.io/en/latest/api/_modules/root/ploomber.DAG.html#ploomber.DAG

I also couldn't find it in the repo so I'd assume it's generated by sphinx